app/ui/browser-modern/sagas/page-sagas.js

```diff
@@ -123,3 +127,16 @@ function* capturePage({ pageId, webview }) {
 
   yield put(ProfileEffects.addCapturedPage(pageId, readerResult));
 }
+
+function* showDownloadNotification({ filename, status }) {
+  let message;
+  const title = `Download ${status === 'success' ? 'completed' : 'failed'}`;
+
+  if (status === 'success') {
+    message = `Download for ${filename} is completed.`;
+  } else if (status === 'error') {
+    message = `Download for ${filename} has failed.`;
+  }
+
+  new Notification(title, { body: message }); //eslint-disable-line
```

Comment:
Oh, apparently this: https://developer.mozilla.org/en-US/docs/Web/API/notification
Very interesting.